operators/pkg/controller/elasticsearch/initcontainer/prepare_fs_script.go

```diff
@@ -98,7 +73,7 @@ var scriptTemplate = template.Must(template.New("").Parse(
 	#  Files persistence #
 	######################
 
-	# Persist the content of bin/, config/ and plugins/
+	# Persist the content of bin/ and config/ 
```

Comment:
I think we don't need to persist bin & config anymore, since was mostly done for additional plugins stuff that can get installed in there.